contrib/jemalloc/src/jemalloc.c

```diff
@@ -2431,7 +2428,7 @@ je_malloc(size_t size) {
 		LOG("core.malloc.exit", "result: %p", ret);
 
 		/* Fastpath success */
-		return BOUND_PTR(ret, zero_size ? 0 : size);
+		return BOUND_PTR(ret, usize);
```

Comment:
You don't mean that. usize is only defined under config_stats || config_prof

Reply:
Indeed. Looks like I should just be calling `sz_index2size(ind)` directly. I suppose we should be storing bounded caps in the cache.

Reply:
That won't give precise bounds any more even in cases where it's totally fine?

Reply:
Agreed